packages/web/src/components/molecules/RunningContext/ContextDisplay.tsx

```diff
@@ -31,9 +31,7 @@ const ContextDisplay: FC<ContextDisplayProps> = ({url, label, tooltip, wrap, tex
   const item = isExternal ? (
     <ExternalLink href={url}>{content || label}</ExternalLink>
   ) : url ? (
-    <a href="#" onClick={navigate}>
-      {content || label}
-    </a>
+    <a onClick={navigate}>{content || label}</a>
```

Comment:
Shouldn't it stay `href="#"` or become button? Otherwise it won't be looking like a link (as only `a[href]` have pointer cursor).

Reply:
the `<a>` tag preserves it's styling (including pointer cursor) even without `href="#"`.  The reason I it's removed, is because it caused a navigation to the root. Went for the alternative solution of `e.preventDefault`

Reply:
The `<a>` without `href` behaves differently, as it's constructed differently in the browser (anchor vs link).

<img width="322" alt="Zrzut ekranu 2023-10-27 o 11 30 11" src="https://github.com/kubeshop/testkube-dashboard/assets/3843526/8faea766-fba5-4bf3-b0cb-d2302b8202c7">

Looks like we are overwriting it with `.ant-typography a` anyway 🙂  I would keep `href` though, to avoid subtle browser behavior changes 👍